Last-Update: 2018-07-04
Forwarded: https://bugs.gnucash.org/show_bug.cgi?id=796732
Bug-Upstream: https://bugs.gnucash.org/show_bug.cgi?id=796732
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: allow setting invoice title.
 Do not hard-code invoice title as "Customer Invoice".

--- a/gnucash/report/business-reports/taxinvoice.eguile.scm
+++ b/gnucash/report/business-reports/taxinvoice.eguile.scm
@@ -458,9 +458,8 @@
            (set! opt-invoice-number-text "Bill number")
            (display-report opt-invoice owner endowner ownertype)
         )
         (begin
-           (set! opt-report-title "Customer Invoice")
            (set! opt-extra-notes "")
            (set! opt-payment-recd-heading "Payment")
            (set! opt-invoice-number-text "Invoice number")
            (display-report opt-invoice owner endowner ownertype)
